lib/chisel-utils.js

```diff
@@ -98,7 +102,13 @@ const serializeImports = (imports, startPathOpt) => {
   return result.join('\n');
 };
 
-const generateBlackBox = ({className, paramsMap, blackboxParamsMap, ioType, ioParamsMap, desiredName}) => {
+const generateBlackBox = ({
+  className,
+  paramsMap,
+  blackboxParamsMap,
+  ioType,
+  ioParamsMap,
+  desiredName}) => {
```

Comment:
Just to double check, there was no actual functional change involved here? I was reading this thing extra carefully just to check if anything changed.

Reply:
yes, no functional change